tests/src/parallel_restore.c

```diff
@@ -604,6 +631,9 @@ static int do_restore(struct opts *opts)
 	ret = scoutfs_parallel_restore_import_super(wri, super, dev_fd);
 	error_exit(ret, "import super %d", ret);
 
+	ret = scoutfs_parallel_restore_add_quota_rule(wri, generate_quota(opts));
```

Comment:
`generate_quota` allocates memory, and returns a pointer to it. The caller should then free this memory after it's been used.